components/CourseOverviewPage.vue

```diff
@@ -1,8 +1,8 @@
 <template>
```

Comment:
All the pages under `pages/learn/course/` uses this component. I think we can create just one `pages/learn/course/[course-id]` and have the content in separate files.

If you think this change is not related with this PR, please, create an issue for refactor it in the future

Reply:
I like this idea very much and I also agree that we should create a new issue for later to handle this: https://github.com/Qiskit/qiskit.org/issues/2983